<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBoxMainInfo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelName">
           <property name="text">
            <string>Название</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="editName"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="buttonSave">
           <property name="text">
            <string>Сохранить шаблон</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="buttonAddBg">
        <property name="text">
         <string>Добавить фон</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
